doc/man3/ASN1_INTEGER_new.pod

```diff
@@ -0,0 +1,43 @@
+=pod
+
+=head1 NAME
+
+ASN1_INTEGER_new, ASN1_INTEGER_free, ASN1_INTEGER_it -
+ASN1_INTEGER allocation functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/asn1.h>
+
+ extern const ASN1_INTEGER_it;
+ ASN1_INTEGER * ASN1_INTEGER_new(void);
+ void ASN1_INTEGER_free(ASN1_INTEGER *a);
+
+=head1 DESCRIPTION
+
+ASN1_INTEGER_new() returns an allocated B<ASN1_INTEGER> structure. Its type
+is undefined.
```

Comment:
Sorry, I don't understand what does it mean.

Reply:
it's a copy/pas issue